lib/identity_config.rb

```diff
@@ -171,6 +171,9 @@ def self.build_store(config_map)
     config.add(:email_registrations_per_ip_track_only_mode, type: :boolean)
     config.add(:email_from, type: :string)
     config.add(:email_from_display_name, type: :string)
+    config.add(:email_reminder_early_benchmark, type: :integer)
+    config.add(:email_reminder_late_benchmark, type: :integer)
+    config.add(:email_reminder_final_benchmark, type: :integer)
```

Comment:
I know it'll make them all really long, but I think we should add `in_person` and `days` to these variable names, like `in_person_email_reminder_early_benchmark_in_days`